snake.c

```diff
@@ -218,7 +218,7 @@ void printField() {
 				printw("  ");
 			}
 			else if ( field[i][j] == -1 ) {
-				printw("O ");
+				printw("S ");
```

Comment:
Why don't you use the constant here? :thinking:

Reply:
Ain't nobody got time for that. Merging.